src/main/java/teammates/common/util/Const.java

```diff
@@ -1546,4 +1546,12 @@ private Const() {
                 + " Raw log message: %1$s";
     }
 
+    public static class EmailTemplateMessages {
```

Comment:
Can consider moving this class to EmailGenerator, since it is only accessed there.